### PR TITLE
Fedora releases guide: add links and shuffle jobs a bit

### PR DESCRIPTION
**Pull request by jpopelka**
correct me if I'm wrong

### CODE REVIEW

File: content/docs/fedora-releases-guide.md [hunk 1]
```diff
@@ -5,17 +5,16 @@ weight: 9
...
-5. Bodhi updates
-
-Doing Fedora releases with Packit means utilising our 4 jobs:
-1. `propose_downstream`
-2. `koji_build`
-3. `bodhi_update`
-4. `pull_from_upstream`
+1. [New upstream release](#new-upstream-release)
+2. [Uploading archive to lookaside cache](#uploading-archive-to-lookaside-cache)
+3. [Updating dist-git content](#updating-dist-git-content)
```
Comment (TomasTomecek) [inline content/docs/fedora-releases-guide.md:10]:
```suggestion
2. [Upload archive to lookaside cache](#upload-archive-to-lookaside-cache)
3. [Update dist-git content](#update-dist-git-content)
```
all other are in passive as well

Comment (TomasTomecek) [reply content/docs/fedora-releases-guide.md:10]: uhhh, can't suggest for unchanged lines -_- up to you, feel free to decline this

**Review by TomasTomecek — APPROVED** [review]
👏👏

**Review by lbarcziova — APPROVED** [review]
thanks!

**Pull request merged**